src/project-manager.ts

```diff
@@ -483,6 +553,20 @@ export class ProjectConfiguration {
                 if (/(^|\/)jsconfig\.json$/.test(self.configFileName)) {
                     options.allowJs = true;
                 }
+
+                if (self.packageJsonFileName) {
+                    try {
+                        let parsedResult = JSON.parse(self.fs.readFile(self.packageJsonFileName));
```

Comment:
Do we santiize `self.packageJsonFileName` so it can't be `/etc/passwd` for example?